src/gateway/WebSocketClient.ts

```diff
@@ -260,15 +260,15 @@ export class WebSocketClient extends EventEmitter {
 
 		if (!this.expectedGuilds?.size) {
 			this.status = Status.READY;
-			// this.emit(ShardEvents.ALL_READY);
+			this.emit(WebSocketEvents.READY);
 
 			return;
 		}
 
 		this.readyTimeout = this.client.setTimeout(() => {
 			this.readyTimeout = undefined;
 			this.status = Status.READY;
-			// this.emit(ShardEvents.ALL_READY, this.expectedGuilds);
+			this.emit(WebSocketEvents.READY, this.expectedGuilds);
```

Comment:
why there is `this.expectedGuilds` as argument here ?

Reply:
See https://discord.com/developers/docs/topics/gateway#ready

Reply:
But it's something specific to the sharding system, so I'm not sure...